.pre-commit-config.yaml

```diff
@@ -0,0 +1,13 @@
+# See https://pre-commit.com for more information
```

Comment:
I'll have a look at that when I have the time but will not merge this PR with this included right now. Please remove that for the time being (and submit as a separate PR, if you wish).